src/main/java/frc/robot/RobotContainer.java

```diff
@@ -80,11 +82,11 @@ public class RobotContainer {
 
     /** The container for the robot. Contains subsystems, OI devices, and commands. */
     public RobotContainer() {
-        setupDriveTrain(); 
+        setupDriveTrain();
         registerPathPlanableCommands();
         setupPathPlaning();
         autoChooser = new AutoChooser2024();
-        autoChooser.addOnValidationCommand(()->CommandUtil.logged(new SetInitOdom(drivetrain,autoChooser)));
+        autoChooser.addOnValidationCommand(()->new SetInitOdom(drivetrain,autoChooser));
```

Comment:
You must have merged from main and you made the wrong choices, all these commands should be wrapped in CommandUtil.logged

Reply:
Did you create a new branch / pull request? What is the difference between this PR and this PR: https://github.com/FRC4048/FRC2024_Java/pull/80? The SP_AmpSubsystem was the branch that got destroyed because another person was commiting to my branch. So I created a new branch without these changes.

Reply:
I have made these changes.